Template/Trigger/CopyTrigger.web.js

```diff
@@ -0,0 +1,20 @@
+(function () {
+    return function (parameters, TagManager) {
+        this.setUp = function (triggerEvent) {
+            TagManager.dom.addEventListener(parameters.document, "copy", function (event) {
+                if (!event.target) {
+                    return;
+                }
+                var target = event.target;
+                var dom = TagManager.dom;
+                triggerEvent({
+                    event: 'mtm.Print',
+                    'mtm.copyElement': target,
```

Comment:
@Findus23 could we also post the copied text and the text length similar to https://gist.github.com/ju1/6fb3294d28a1461e647349c69a4e30e3#file-copytextrecipe-json-L101 ? The length allows users to filter a specific length like > 30 & < 300 characters which might be eg copied for a tweet.